packages/types/src/clerk.ts

```diff
@@ -227,6 +228,8 @@ export interface Clerk {
   /** Current User. */
   user: UserResource | null | undefined;
 
+  __internal_state: State | undefined;
```

Comment:
Can we add a TSDoc here to explain why this was introduced and how we plan to use it?
The comment is meant for the Clerk team and any users, so it must be marked as `@experimental`